localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -13,3 +13,39 @@
  high_overlord_female:0 "High Overlord"
  overlord_male:0 "Overlord"
  overlord_female:0 "Overlord"
+
+### Wastewander ###
+# Feudal, Clan, or Tribal
+ baron_feudal_male_wastewander:1 "Robber Baron"
+ baron_feudal_female_wastewander:1 "Robber Baroness"
+ barony_feudal_wastewander:1 "Barony"
+ count_feudal_male_wastewander:0 "Sheikh"
+ count_feudal_female_wastewander:1 "Shaykhah"
+ county_feudal_wastewander:0 "Sheikhdom"
+ duke_feudal_male_wastewander:1 "Emir"
+ duke_feudal_female_wastewander:1 "Emira"
+ duchy_feudal_wastewander:0 "Emirate"
+ king_feudal_male_wastewander:0 "Sultan"
+ king_feudal_female_wastewander:0 "Sultana"
+ kingdom_feudal_wastewander:0 "Sultanate"
+ emperor_feudal_male_wastewander:1 "Caliph"
+ emperor_feudal_female_wastewander:1 "Calipha"
+ empire_feudal_wastewander:1 "Caliphate"
+
+### Northern Human ###
+# Feudal, Clan, or Tribal
+ duke_feudal_male_northern:0 "Grand Prince"
+ duke_feudal_female_northern:0 "Grand Princess"
+ duchy_feudal_northern:0 "Grand Principality"
+ emperor_feudal_male_northern:0 "Tsar"
+ emperor_feudal_female_northern:0 "Tsaritsa"
+ empire_feudal_northern:0 "Tsardom"
+
+### Kul Tiras ###
+# Feudal, Clan, or Tribal
+ king_feudal_male_tirassian:0 "Lord Admiral"
+ king_feudal_female_tirassian:0 "Lord Admiral"
+ kingdom_feudal_tirassian:0 "Lord Admiralty"
+ emperor_feudal_male_tirassian:1 "Grand Admiral"
+ emperor_feudal_female_tirassian:1 "Grand Admiral"
+ empire_feudal_tirassian:1 "Grand Admiralty"
```

Comment:
Kul'tiras is only named "Admiralty of Kul'tiras" and not "Lord Admiralty of Kul'tiras". So I suggest that it should be "Admiralty of" just for the kingdoms.

![image](https://user-images.githubusercontent.com/46578034/110052206-801dde80-7d57-11eb-96c4-e96b900d3568.png)

Reply:
> Kul'tiras is only named "Admiralty of Kul'tiras" and not "Lord Admiralty of Kul'tiras". So I suggest that it should be "Admiralty of" just for the kingdoms.
> 
> ![image](https://user-images.githubusercontent.com/46578034/110052206-801dde80-7d57-11eb-96c4-e96b900d3568.png)

That's an empire title. So it's a possible title of Kul Tiras if they become more powerful.